doc/ss_logger.rst

```diff
@@ -0,0 +1,144 @@
+****************
+SmartSim Logging
+****************
+========
+Overview
+========
+SmartSim supports experiment tracking with logging functionality
+using the SmartSim log module. The logger, supported by Python logging, enables
+monitoring the experiment at runtime by allowing users to print messages **to stdout**
+and/or **to file** from within the Python driver script. The SmartSim logger allows users to categorize
+messages by severity level. Users may instruct the SmartSim to print
```

Comment:
I think `Users may instruct the SmartSim` should be `Users may instruct SmartSim`